src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DriverComponentSpec.scala

```diff
@@ -14,12 +14,15 @@ class DriverComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
   //   validateAttributeName
   //   createBatches
 
+  implicit val getWorkflowRecord = GetResult { r => WorkflowRecord(r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<) }
+
+  val selectAllFromWorkflow = "SELECT ID, EXTERNAL_ID, SUBMISSION_ID, STATUS, STATUS_LAST_CHANGED, ENTITY_ID, record_version FROM WORKFLOW"
+
   "DriverComponent" should "test concatSqlActions" in withDefaultTestDatabase {
-    implicit val getWorkflowRecord = GetResult { r => WorkflowRecord(r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<) }
 
-    val select = sql"SELECT * FROM WORKFLOW "
```

Comment:
nevermind, i read that as "may choose the **rows**" instead of "may choose the columns. ignore.